test/commands/test_alias.rb

```diff
@@ -1,7 +1,45 @@
 require 'minitest/autorun'
+require_relative '../test__helper'
+require_relative '../fake_home'
+require_relative '../../lib/zold/commands/alias'
 
-# @todo #279:30min Write unit tests for the alias command.
-#  It should test all the subcommands (set, remove and show)
-#  and any options.
 class TestAlias < Minitest::Test
+  # alias set <wallet> <alias>
+  def test_set_writes_alias_to_the_alias_file
+    skip
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      Zold::Alias.new(wallets: home.wallets, log: test_log).run(%W[set #{wallet.id} my-alias])
+      aliases = File.read(File.join(home.dir, 'aliases')).split(' ')
+      assert_equal aliases, %W[my-alias #{wallet.id}]
+    end
+  end
+
+  # alias remove <alias>
+  def test_remove_removes_the_alias_from_the_alias_file
+    skip
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      cmd = Zold::Alias.new(wallets: home.wallets, log: test_log)
+      cmd.run(%W[set #{wallet.id} my-alias])
+      aliases = File.read(File.join(home.dir, 'aliases')).split(' ')
+      assert_equal aliases, %W[my-alias #{wallet.id}]
+      cmd.run(%w[remove my-alias])
+      assert_empty aliases
```

Comment:
@botanicus notice how you don't read the file after you've updated it. If you extract `alases`, it'll work though.